package.json

```diff
@@ -39,95 +39,97 @@
+    "@angular/compiler-cli": "^14.3.0",
+    "@angular/language-service": "^14.3.0",
+    "@commitlint/cli": "^17.6.7",
+    "@commitlint/config-angular": "^17.6.7",
+    "@commitlint/config-conventional": "^17.6.7",
+    "@commitlint/prompt": "^17.6.7",
+    "@compodoc/compodoc": "^1.1.21",
+    "@semantic-release/github": "^9.0.4",
+    "@semantic-release/npm": "^9.0.2",
+    "@storybook/addon-a11y": "^7.2.2-alpha.0",
+    "@storybook/addon-essentials": "^7.2.2-alpha.0",
+    "@storybook/addons": "^7.2.2-alpha.0",
+    "@storybook/angular": "^7.2.2-alpha.0",
+    "@storybook/theming": "^7.2.2-alpha.0",
     "@stylelint/postcss-css-in-js": "^0.38.0",
     "@stylelint/postcss-markdown": "^0.36.2",
-    "@types/jasmine": "^4.3.1",
+    "@types/jasmine": "^4.3.5",
     "@types/jasminewd2": "^2.0.10",
     "@types/node": "^18.11.18",
-    "@typescript-eslint/eslint-plugin": "5.49.0",
-    "@typescript-eslint/parser": "5.49.0",
-    "@typestrong/ts-mockito": "^2.6.4",
-    "autoprefixer": "^10.4.13",
+    "@typescript-eslint/eslint-plugin": "5.62.0",
+    "@typescript-eslint/parser": "5.62.0",
+    "@typestrong/ts-mockito": "^2.6.6",
+    "autoprefixer": "^10.4.14",
     "codelyzer": "^6.0.2",
     "color-convert": "^2.0.1",
-    "commitizen": "^4.2.5",
+    "commitizen": "^4.3.0",
     "copy-webpack-plugin": "^11.0.0",
     "cz-conventional-changelog": "^3.3.0",
-    "eslint": "^8.23.0",
+    "eslint": "^8.46.0",
     "eslint-plugin-html": "^7.1.0",
-    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-import": "^2.28.0",
     "eslint-plugin-jasmine": "^4.1.3",
-    "eslint-plugin-unused-imports": "^2.0.0",
-    "fs-extra": "^10.1.0",
+    "eslint-plugin-unused-imports": "^3.0.0",
+    "fs-extra": "^11.1.1",
     "husky": "^8.0.3",
-    "jasmine-core": "^4.5.0",
+    "jasmine-core": "^5.1.0",
     "jasmine-marbles": "^0.9.2",
     "jasmine-spec-reporter": "^7.0.0",
-    "karma": "^6.4.1",
-    "karma-chrome-launcher": "^3.1.1",
+    "karma": "^6.4.2",
+    "karma-chrome-launcher": "^3.2.0",
     "karma-coverage-istanbul-reporter": "^3.0.3",
     "karma-jasmine": "^5.1.0",
-    "karma-jasmine-html-reporter": "^2.0.0",
+    "karma-jasmine-html-reporter": "^2.1.0",
     "license-checker": "^25.0.1",
-    "lint-staged": "^13.0.4",
-    "mini-css-extract-plugin": "^2.7.2",
-    "ng-mocks": "^14.4.0",
+    "lint-staged": "^13.2.3",
+    "mini-css-extract-plugin": "^2.7.6",
+    "ng-mocks": "^14.11.0",
     "ng-packagr": "^14.2.2",
-    "postcss": "^8.4.19",
+    "postcss": "^8.4.27",
     "postcss-scss": "^4.0.6",
-    "prettier": "^2.8.1",
+    "prettier": "^3.0.1",
     "pretty-quick": "^3.1.3",
     "protractor": "^7.0.0",
-    "sass": "^1.56.1",
-    "semantic-release": "^19.0.5",
-    "stylelint": "^14.15.0",
-    "stylelint-config-recommended": "^9.0.0",
-    "stylelint-selector-bem-pattern": "^2.1.1",
-    "svg-to-ts": "^8.9.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "remark-gfm": "^3.0.1",
```

Comment:
This package had to be added as MDX doesn't support GitHub Flavoured Markdown anymore: https://storybook.js.org/docs/react/writing-docs/mdx#lack-of-github-flavored-markdown-gfm